docs/faq.md

```diff
@@ -413,6 +413,34 @@ For reproducibility purposes, Jib sets the creation time of the container images
 jib.container.useCurrentTimestamp = true
 ```
 
+#### Please tell me more about reproducibility!
+
+_Reproducible_ means that building and rebuilding with Jib should
```

Comment:
If we do want this (I think it looks good, it's a nice explanation) I think we should remove all these extra newlines, or it could potentially mess with the formatting for different screen sizes.

Reply:
OK. I'll leave @TadCordle for wording suggestion.